tests/vcr_python_wrapper.py

```diff
@@ -121,9 +146,24 @@ def filter_request_manifest(request):
     return request
 
 
+def filter_request_ldap(request):
+    request.uri = urlunparse(urlparse(request.uri)._replace(netloc="foreman.example.org"))
+    if request.body is not None:
+        try:
+            json_body = json.loads(request.body)
+        except json.JSONDecodeError:
```

Comment:
https://stackoverflow.com/a/35214768
Seems like we should check for `ValueError` instead, to make py27 happy.